core/math/basis.cpp

```diff
@@ -123,17 +123,20 @@ bool Basis::is_rotation() const {
 	return Math::is_equal_approx(determinant(), 1, UNIT_EPSILON) && is_orthogonal();
 }
 
+#ifdef MATH_CHECKS
+// This method is only used once, in diagonalize. If it's desired elsewhere, feel free to remove the #ifdef.
 bool Basis::is_symmetric() const {
 
-	if (!Math::is_equal_approx_ratio(elements[0][1], elements[1][0], UNIT_EPSILON))
+	if (!Math::is_equal_approx(elements[0][1], elements[1][0]))
```

Comment:
The previous version using `UNIT_EPSILON` was a lot more lenient since it defaults to 0.001 *and* it was a ratio. This will likely bring back the "false positive" !symmetric errors that led @reduz to implement a lenient `UNIT_EPSILON` in the first place.

So you should discuss this with him.

Reply:
Related discussion: https://github.com/godotengine/godot/issues/37241